packages/flutter/test/material/data_table_test.dart

```diff
@@ -918,4 +919,115 @@ void main() {
     boxDecoration = tableRow.decoration as BoxDecoration;
     expect(boxDecoration.border.bottom.width, thickness);
   });
+
+  testWidgets('DataTable column heading cell - no sort', (WidgetTester tester) async {
+    Widget buildTable({ int sortColumnIndex, bool sortEnabled = true }) {
+      return DataTable(
+        sortColumnIndex: sortColumnIndex,
+        columns: <DataColumn>[
+          DataColumn(
+            label: const Expanded(child: Center(child: Text('Name'))),
+            tooltip: 'Name',
+            onSort: sortEnabled ? (_, __) {} : null,
+          ),
+        ],
+        rows: const <DataRow>[
+          DataRow(
+            cells: <DataCell>[
+              DataCell(Text('A long desert name')),
+            ],
+          ),
+        ]
+      );
+    }
+
+    await tester.pumpWidget(MaterialApp(
+      home: Material(child: buildTable(
+        sortEnabled: false,
+      )),
+    ));
+
+    {
```

Comment:
What are these brackets enclosing? Here and below.

Reply:
These brackets are only to scope `nameText` and `nameCell` between subtests so that I don't have to either use `nameText1` `nameText2` etc, or declare 5 empty variables at the start of the test.

Reply:
Good to know! Thanks!